backend/models/test.model.ts

```diff
@@ -123,8 +125,26 @@ const TestSchema: Schema = new Schema(
       type: Number,
       required: true,
     },
+    curriculumCountry: {
+      type: String,
+      required: true,
+    },
+    curriculumRegion: {
+      type: String,
+      required: true,
+    },
+    assessmentType: {
+      type: String,
+      required: true,
+      enum: ["Beginning", "End"],
+    },
+    status: {
+      type: String,
+      required: true,
+      enum: ["Draft", "Published", "Archived", "Deleted"],
+    },
```

Comment:
I think we can define both `assessmentType` and `status` enums above! And then no need to define a separate type. i.e line 54 in `backend/types.ts`. For an example of what I mean, see [definition](https://github.com/uwblueprint/jump-math/blob/bc6ee0458db1270225a12646af94cfe290435a97/backend/models/testSession.model.ts#L6) of `GradingStatus`!